fastlane_core/lib/fastlane_core/keychain_importer.rb

```diff
@@ -12,8 +12,27 @@ def self.import_file(path, keychain_path, keychain_password: "", certificate_pas
       command << " -T /usr/bin/security"
       command << " 1> /dev/null" unless output
 
-      Helper.backticks(command, print: output)
+      UI.command(command) if output
+      Open3.popen3(command) do |stdin, stdout, stderr, thrd|
+        UI.command_output(stdout.read.to_s) if output
 
+        # Set partition list only if success since it can be a time consuming process if a lot of keys are installed
+        if thrd.value.success?
+          set_partition_list(path, keychain_path, keychain_password: keychain_password, certificate_password: certificate_password, output: output)
+        else
+          # Output verbose if file is already installed since not an error otherwise we will show the whole error
+          err = stderr.read.to_s.strip
+          if err.include?("SecKeychainItemImport") && err.include?("The specified item already exists in the keychain")
+            type = File.extname(".p12") ? "Key" : "Certificate"
```

Comment:
`File.extname(".p12")` always returns `""` which is always true. Is the intended parameter `path`?

Reply:
Yes, that definitely looks like a bug

Reply:
Oops! lol, good catch

Reply:
I should add a test for that

Reply:
Fixed!! ^ I just removed that type completely